core/src/main/scala/com/rallyhealth/vapors/core/dsl/factfilter/Types.scala

```diff
@@ -40,7 +40,8 @@ private[dsl] trait Types {
   /**
     * An expression that terminates into a boolean, used for making a conditional query or filter.
     *
-    * Useful for defining sub-expressions to [[whereAnyFactHas]], [[whereAllFactsHave]], [[exists]], [[all]], etc.
+    * Useful for defining sub-expressions to [[WhereFactsExpBuilder.whereAllFacts]],
+    * [[WhereFactsExpBuilder.whereAnyValue]], [[exists]], [[all]], etc.
```

Comment:
This change still didn't satisfy the ScalaDoc compiler, but it fixed it for IntelliJ IDEA.